client/conf/server.properties.in

```diff
@@ -24,6 +24,9 @@ context.path=/client
 # The HTTP port to be used by the management server
 http.port=8080
 
+# Max inactivity time in minutes for the session
+session.timeout=10
```

Comment:
@marcaurele Can we bump the default to 30 minutes? I suppose 10 minutes might be too low, or maybe we can hit the sweet spot at 20mins?

Reply:
I agree that 10 minutes is very short. I would prefer 30